.github/workflows/publish_image.yml

```
@@ -144,4 +149,5 @@ jobs:
     uses: ./.github/workflows/mirror.yml
     with:
       version: ${{ needs.settings.outputs.image_version }}
-    secrets: inherit
+    secrets:
+      PROD_AWS_ROLE: ${{ secrets.PROD_AWS_ROLE }}
```

actionlint (1.7.12)

[error] secret "PROD_AWS_ROLE" is not defined in "./.github/workflows/mirror.yml" reusable workflow. no secret is defined

(workflow-call)